package.json

```diff
@@ -18,12 +18,13 @@
         "cypress:run": "REACT_APP_NODE_ENV=test yarn cypress run"
     },
     "devDependencies": {
-        "@dhis2/cli-app-scripts": "^10.2.0",
+        "@dhis2/cli-app-scripts": "^10.3.9",
```

Comment:
Upgraded this mostly to fix `yarn test --testRegex`. Previous version split the string per character, so it didn't work as expected.